tpchgen/src/generators.rs

```diff
@@ -433,6 +443,11 @@ impl<'a> PartGenerator<'a> {
         }
     }
 
+    /// Return the row count for the given scale factor and generator part count
```

Comment:
I refactored out the row count calculation into a function that can be called from outside (needed to calculate how many parts for each table)